### PR TITLE
Create du00.cc.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/du00.cc.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<ruleset name="du00.cc">
+
+	<target host="m.du00.cc" />
+
+	<target host="www.du00.cc" />
+
+	<!--	MCB:The text could not show.	-->
+
+		<exclusion pattern="^http://www\.du00\.cc/read/\d+/\d+/\d+\.html" />
```
Comment (J0WI) [inline src/chrome/content/rules/du00.cc.xml]: same on m.

File: src/chrome/content/rules/du00.cc.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  		<test url="http://www.du00.cc/read/39/39898/9502924.html" />
+  		<test url="http://www.du00.cc/read/47/47751/11503878.html" />
+  		<test url="http://www.du00.cc/read/67/67925/16855700.html" />
+  		<test url="http://www.du00.cc/read/70/70200/17351417.html" />
+
+		<exclusion pattern="^http://www\.du00\.cc/t/t\.php\?id=" />
```
Comment (J0WI) [inline src/chrome/content/rules/du00.cc.xml]: same on m.
Please strictly use tabs to indent the test urls.

**Comment by J0WI** [comment]
Could you also add a comment about ^?

File: src/chrome/content/rules/du00.cc.xml (hunk 2 above)
Comment (renyouguo) [inline src/chrome/content/rules/du00.cc.xml]: m. works very well for me on both PC and Android.

Comment (J0WI) [inline src/chrome/content/rules/du00.cc.xml]: m. is currently down for me, but I thought that I also saw mixed active content there.

Comment (renyouguo) [inline src/chrome/content/rules/du00.cc.xml]: @J0WI I have add comments for them.

**Pull request merged**